test/images/agnhost/webhook/pods.go

```diff
@@ -77,6 +80,32 @@ func admitPods(ar v1.AdmissionReview) *v1.AdmissionResponse {
 }
 
 func mutatePods(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	shouldPatchPod := func(pod *corev1.Pod) bool {
+		if pod.Name != "webhook-to-be-mutated" {
+			return false
+		}
+		return !hasContainer(pod, "webhook-added-init-container")
```

Comment:
hasContainer is checking spec.Containers, but we will inject to spec.InitContainers

Reply:
Good catch. Fixed.